biome.jsonc

```diff
@@ -44,34 +52,39 @@
 			"trailingComma": "all",
 			"quoteProperties": "preserve",
 			"quoteStyle": "double",
-			"bracketSpacing": true
-		}
+			"bracketSpacing": true,
+		},
 	},
 	"json": {
 		"formatter": {
-			"indentStyle": "tab"
-		}
+			"indentStyle": "tab",
+		},
 	},
 	"overrides": [
 		{
-			"include": ["**/*.jsonc", "**/.vscode/*.json", "**/tsconfig*.json", "**/api-extractor*.json"],
+			"include": [
+				"**/*.jsonc",
+				"**/.vscode/*.json",
+				"**/tsconfig*.json",
+				"**/api-extractor*.json",
+			],
 			"json": {
 				"parser": {
 					"allowComments": true,
-					"allowTrailingCommas": true
+					"allowTrailingCommas": true,
 				},
 				"formatter": {
-					"trailingCommas": "all"
-				}
-			}
+					"trailingCommas": "all",
+				},
+			},
 		},
 		{
 			"include": ["**/package.json"],
 			"json": {
 				"formatter": {
-					"lineWidth": 80
-				}
-			}
-		}
-	]
+					"lineWidth": 80,
```

Comment:
Tangent: not sure how this rule could be relevant.... if something goes past 80 chars because it's a string, or the json structure requires a ton of indentation... what can a formatter do about that?

Reply:
I updated the setting to match my original intent (formatting like npm) and added a comment explaining why the settings
are used.